tests/regression_tests/mgxs_library_mesh/results_true.dat

```diff
@@ -180,38 +180,38 @@
 3      2  2  1        1         1   total  0.031336  0.001614
    mesh 1               group in nuclide    mean std. dev.
         x  y       surf                                   
-0       1  1  x-min out        1   total  0.0000  0.000000
-1       1  1   x-min in        1   total  0.0000  0.000000
-2       1  1  x-max out        1   total  0.2738  0.093735
 3       1  1   x-max in        1   total  0.1892  0.011302
-4       1  1  y-min out        1   total  0.0000  0.000000
-5       1  1   y-min in        1   total  0.0000  0.000000
-6       1  1  y-max out        1   total  0.2358  0.041204
+2       1  1  x-max out        1   total  0.2738  0.093735
```

Comment:
any reason the order of the bins is flipped ?

Reply:
Can you please explain what you mean? What is flipped? It all seems to be in proper order to me.

Reply:
It goes 3,2,1,0,7,6,5,4. It also goes x-max before x-min and in before out.
The previous order seems to have been x-min before x-max and out before in.

Reply:
In my previous commits, the order was arbitrary and in fact was inconsistent across different python versions in the test results. I was able to fix the order using sort_values(by= ... 'surf' ...), so the order is the one that will stick. It doesn't really matter if min is before max, as long as it is consistent across systems. 
The numbering on the left-hand-side doesn't seem to matter. What matters is imposing the order in the mesh, first by x, then y, then z (then 'surf' in the case of current). If you notice the other results also have the numbers listed in an unordered way. That's because the order is imposed in sort_values() in mgxs.

Reply:
Ok agreed sort_values() is the way to go.